jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -692,12 +730,12 @@ private static DiagnosticProperties doCheckpointJVM(String diagnosticCommand) {
 		commandTable.put(DIAGNOSTICS_LOAD_JVMTI_AGENT, DiagnosticUtils::loadJVMTIAgent);
 		helpTable.put(DIAGNOSTICS_LOAD_JVMTI_AGENT, DIAGNOSTICS_LOAD_JVMTI_AGENT_HELP);
 
-/*[IF CRAC_SUPPORT]*/
-		if (InternalCRIUSupport.isCRaCSupportEnabled()) {
+/*[IF CRAC_SUPPORT | CRIU_SUPPORT]*/
```

Comment:
lets not combine CRac support with CRIUSupport

Reply:
Lets make them separate commands. You can only use one of them at a time currently.

Reply:
Sure, added `CRIU.checkpoint` for CRIU-enabled JVM and kept `JDK.checkpoint` for CRaC.